### ISSUE TITLE
Have you reproduced the accuracy on Something dataset?

### ISSUE THREAD

**Issue by nemonameless** [Pika7ma/Temporal-Shift-Module#1]

**Comment by PingchuanMa** [comment]
It is still on the way. I will publish my results as soon as I make it.

**Comment by nemonameless** [comment]
@Pika7ma  Hi, have you  first pre-train on Kinetics ?  Otherwise, it seems that the accuracy cannot be reproduced. I just used imagenet-pretrained resnet50 and  fixed the bug in https://github.com/Pika7ma/Temporal-Shift-Module/blob/master/tsm_util.py#L14  but only got  41 on something-v1.  How about you?

**Comment by PingchuanMa** [comment]
@nemonameless Yes it should be pre-trained on Kinetics following the steps of original paper.

**Comment by Haijunlv** [comment]
@Pika7ma did you reproduce  kinetics top1 72.5% at 16Frames?  I followed tsn based source code(a little diffent at init weight).  trained 45 epoch  with tran param from paper. But only get 70.436% top1.

**Comment by PingchuanMa** [comment]
@Haijunlv Actually no. I am currently doing experiments using 8-frames. I'll try it soon thanku for the information.

**Comment by tzzcl** [comment]
> @Pika7ma did you reproduce kinetics top1 72.5% at 16Frames? I followed tsn based source code(a little diffent at init weight). trained 45 epoch with tran param from paper. But only get 70.436% top1.

Hi, I'm interested in that do you use pre-trained resnet50 on ImageNet as init params? In the paper they don't claim for this but I guess it is important.

**Comment by Haijunlv** [comment]
@tzzcl yes. I use tsn codebase to reproduce. But still have 2% gap with the paper. Maybe pretrained weight is important. But lateset paper by kaiming said pretrained is not necessary. Anyway pretained model save time to converge.

**Comment by tzzcl** [comment]
> @tzzcl yes. I use tsn codebase to reproduce. But still have 2% gap with the paper. Maybe pretrained weight is important. But lateset paper by kaiming said pretrained is not necessary. Anyway pretained model save time to converge.

Hi, I also use the tsn codebase and pretrained resnet50 to reimplement K=8 results on Kinetics. But there is also 2% gap (about 68.8% with my own implementation). Maybe we can only wait for the official code.

**Comment by gongbudaizhe** [comment]
@nemonameless May I ask what are your configurations to get 41% on something v1 ? e.g. Batch size, learning rate schedule

**Comment by sKamiJ** [comment]
hi guys, have you reproduced the accuracy on the Something-Something dataset based on the official code at https://github.com/mit-han-lab/temporal-shift-module ?
